zip-0208.rst

```diff
@@ -0,0 +1,131 @@
+::
+
+  ZIP: 208
+  Title: Shorter Block Target Spacing
+  Author: Simon Liu <simon@z.cash>
+          Daira Hopwood <daira@z.cash>
+  Category: Consensus
+  Created: 2018-01-10
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as
+described in RFC 2119. [#RFC2119]_
+
+The terms "block chain", "consensus rule change", "branch", and "network upgrade" are to be
+interpreted as defined in [#zip-0200]_.
+
+The term "block target spacing" means the time interval between blocks targeted by the difficulty
+adjustment algorithm in a given branch. It is normally measured in seconds.
+
+
+Abstract
+========
+
+This proposal specifies a change in the block target spacing, to take effect in the Blossom
+network upgrade [#zip-0206]_.
+
+The emission schedule of mined ZEC will be approximately the same in terms of time, but this
+requires the emission per block to be adjusted to take account of the changed block target spacing.
+
+
+Motivation
+==========
+
+- Faster finalization
```

Comment:
where finalization is generally understood to be the number of confirmations a transaction has received before it is considered to be settled [#krakensupport]_.  As a rule of thumb, the higher the number of confirmations, the lower the probability that an attacker can invalidate the transaction [#doublespend]_.  Faster blocker times are said to help achieve the requisite security margin in a shorter time period [#slowfastblocks]_.